src/plugins/workspace/public/components/workspace_form/utils.ts

```diff
@@ -65,23 +61,14 @@ export const convertApplicationsToFeaturesOrGroups = (
 ) => {
```

Comment:
It seems the applications only need id, title and category. Can we remove navLinkStatus and chromeless?